src/obfl-specification.html

```diff
@@ -343,13 +343,56 @@ <h3 id="L5025">Auxiliary Flows</h3>
 of these containers, starting from 1 each time.</p>
 
 <h3 id="L5023">Simplex</h3>
-<p class="markup">In a sequence where duplex is not enabled, only the front side of sheets are counted and paginated. 
-The back side of sheets are neither counted nor paginated.</p>
+<p class="markup">In a sequence where duplex is not enabled, only the front side
+  of sheets are counted and paginated. The back side of sheets are neither
+  counted nor paginated.</p>
 
 <h3 id="L5024">Duplex</h3>
-<p class="markup">In a sequence where duplex is enabled, both sides of sheets are counted and paginated.
-However, if the last sheet of a duplex sequence only has content on the front side, 
-the back side is counted, but not <em>paginated</em>.</p>
+<p class="markup">In a sequence where duplex is enabled, both sides of sheets
+  are counted and paginated. However, if the last sheet of a duplex sequence
+  only has content on the front side, it depends on the values of the optional
+  attributes break-before and break-after on sequence, dynamic-sequence and
+  toc-sequence whether there is a back side, and if so, whether it is counted or
+  paginated. The default behavior is that there is a back side and it is counted
+  but not <em>paginated</em>.</p>
+
+<ins datetime="20200612T15:00:00">The last page of a sequence where duplex is
+enabled, may not be paginated and may not be counted.</ins>
+<p></p>
+
+<div class="markup">The just-mentioned break-before and break-after attributes
+  insert a break before or after the sequence, respectively. Depending on the
+  specific element, some of these values are supported:
+  <dl>
+    <dt>'volume'</dt>
+      <dd>continue in a new volume with or after this sequence,
+        respectively.</dd>
```

Comment:
I am not sure about this myself, because 'volume' is not an allowed value in any `break-after`.

Reply:
Since this is a reference document, which should be aimed at finding information efficiently and not so much at providing a pleasant reading experience, I would opt for simply explaining the allowed values for each element separately, thereby repeating stuff where needed. Especially in this case where the allowed values, and even their meanings, depend on the context. I think this list, at least the way it is currently worded, does more bad than good.

I see what you're trying to do here though. In the paragraph below you explain how the attributes interact, and for this it is indeed required to already have a basic understanding of which values exist. Not sure how to solve this best.

Reply:
Also, nothing is stopping us from adding `volume` to the `break-after` values. Would be good for consistency.

Reply:
I've added `volume` to `break-after`.